tests/test_type_levelist.cc

```diff
@@ -51,12 +52,14 @@ CASE("test_metkit_expand_levelist") {
                         {"-10", "-9", "-8", "-7", "-6", "-5", "-4", "-3", "-2", "-1", "0", "1", "2"});
     //  by > 0 && from < to
     assertTypeExpansion("levelist", {"4", "to", "20", "by", "4"}, {"4", "8", "12", "16", "20"});
+    assertTypeExpansion("levelist", {"4", "to", "18", "by", "4"}, {"4", "8", "12", "16"});
     // by > 0 && from > to
     EXPECT_THROWS(assertTypeExpansion("levelist", {"20", "to", "4", "by", "4"}, {"4", "8", "12", "16", "20"}));
     // by = 0
-    assertTypeExpansion("levelist", {"4", "to", "20", "by", "0"}, {"4"});
-    assertTypeExpansion("levelist", {"-1", "to", "2", "by", "0"}, {"-1"});
+    EXPECT_THROWS(assertTypeExpansion("levelist", {"4", "to", "20", "by", "0"}, {"4"}));
```

Comment:
Can we do EXPECT_THROWS_AS in tests, where possible, to identify that we are reporting the correct type of error?